### PR TITLE
Feature/stripe

### PR DESCRIPTION
**Pull request by Boyok1337**
integrated stripe to work with payment

### CODE REVIEW

**Review by oleksandra-shershen — APPROVED** [review]
LGTM!

**Review by Gilgumish — APPROVED** [review]
GJ!

File: docker-compose.yaml [hunk 1]
```diff
@@ -5,11 +5,12 @@ services:
     env_file:
       - .env
     ports:
-      - "8001:8000"
+      - "8000:8000"
```
Comment (VitalyBashkiser) [inline docker-compose.yaml:8]: Shouldn't the left port be raised by 1 unit ?

File: docker-compose.yaml [hunk 2]
```diff
@@ -5,11 +5,12 @@ services:
...
     volumes:
       - ./:/app
     command: >
       sh -c "python manage.py wait_for_db &&
+            python manage.py makemigrations &&
```
Comment (VitalyBashkiser) [inline docker-compose.yaml:13]: Good solution to add makemigrations

**Review by VitalyBashkiser — APPROVED** [review]
LGTM

**Pull request merged**